_pages/how-we-work/leading-projects.md

```diff
@@ -17,6 +17,7 @@ Here is some of what _success_ includes:
 * Establishing a healthy team with positive working relationships
 * The strategic direction of the project (including alignment with a portfolio, when relevant)
 * Managing work so that team members can be fully billable
+* Establishing a system to fairly rotate administrative or clerical tasks between teammates
```

Comment:
I'm not sure I would put "establishing a system to rotate tasks" quite at the level of "core definition of project success," if that makes sense. (This list is at a much higher level, and I'd argue that rotating tasks is a tactical part of "establishing a healthy team with positive working relationships.") Let's cut it from this list but keep the section below.